third_party/move/move-model/src/ast.rs

```diff
@@ -1209,7 +1209,7 @@ impl Operation {
         )
     }
 
-    /// Whether the operation alllows to take reference parameters instead of values. This applies
+    /// Whether the operation allows to take reference parameters instead of values. This applies
```

Comment:
Given that you're changing this, let's make it actually grammatical.  As written, the verb "to take" is lacking a subject in the subclause.  Two options:
- Whether the operation allows reference parameters instead of values.
- Whether the operations can take reference parameters instead of values.

Reply:
I don't mind about such English issues, there are grammatical errors all over the place. I do not have time to fix this.